src/pip/_internal/cli/req_command.py

```diff
@@ -58,11 +58,10 @@ def _create_truststore_ssl_context() -> Optional["SSLContext"]:
         return None
 
     try:
-        import truststore
-    except ImportError:
+        from pip._vendor import truststore
+    except ImportError as e:
         raise CommandError(
-            "To use the truststore feature, 'truststore' must be installed into "
-            "pip's current environment."
+            f"The truststore feature is unavailable: {str(e)}"
```

Comment:
```suggestion
            f"The truststore feature is unavailable: {e}"
```

f-string stringifies automatically